src/client/cosmos.ts

```diff
@@ -48,7 +50,13 @@ export default class CosmosDbClient {
     retryOptions: TimeoutsOptions = INTERNAL_RETRY_OPTIONS
   ) {
     try {
-      const client = new CosmosClient({ endpoint: account, key });
+      const connectionPolicy = new ConnectionPolicy();
+      connectionPolicy.RetryOptions = new RetryOptions(0);
```

Comment:
LGTM, you might want to set a reasonable `requestTimeout` as well, the default value from library is 60s which doesn't quite make sense.